UnityProject/Assets/Scripts/HealthV2/Living/MedicalChemistry/BodyHealthEffect.cs

```diff
@@ -23,7 +23,7 @@ public class BodyHealthEffect : MetabolismReaction
 	public bool CanOverdose = true;
 
 	[ShowIf(nameof(CanOverdose))] public float ConcentrationBloodOverdose = 20f;
-	[ShowIf(nameof(CanOverdose))] public float OverdoseDamageMultiplier = 1;
+	[ShowIf(nameof(CanOverdose))] public float OverdoseDamageMultiplierNew = 1;
```

Comment:
why did you rename this?

Reply:
part of OD balancing

Reply:
ok but how changing the name and nothing else about it is balancing OD?

Reply:
it resets the multiplier on the SOs